examples/reference/custom_components/Viewer.md

```diff
@@ -0,0 +1,258 @@
+# `Viewer`
+
+`Viewer` simplifies the creation of custom Panel components using Python and Panel components only.
```

Comment:
If we had created this component today, it should be called `PyComponent` for consistency reasons. I believe its worth considering doing that - though keep `Viewer` around but deprecate it.